vta/src/runtime.cc

```diff
@@ -1045,6 +1104,7 @@ class CommandQueue {
     if (insn_queue_.count() == 0) return;
     // Synchronization for the queues
     uop_queue_.AutoReadBarrier();
+    insn_queue_.RewriteSetUopKernelPhyAddr(uop_queue_.GetPhyAddrMap());
     insn_queue_.AutoReadBarrier();
```

Comment:
With this change, the `AutoReadBarrier` will trigger a dma transfer. The rewriting was removed today since we are now pre-allocating the FPGA readable buffer instead of doing it on the fly.

Yes, when we do `VTADeviceRun()` we need the buffer to be allocated and mapped to a physical address.